src/store/flow/block-types/MobilePrimitives_SelectManyResponseBlockStore.ts

```diff
@@ -43,58 +51,23 @@ const actions: ActionTree<IEmptyState, IRootState> = {
     }
   },
 
-  handleBranchingTypeChangedToUnified({dispatch}, {block}: { block: ISelectManyResponseBlock }) {
-    dispatch('flow/block_convertExitFormationToUnified', {
-      blockId: block.uuid,
-      test: formatTestValueForUnifiedBranchingType(block),
-    }, {root: true})
-  },
-
   async createWith({getters, dispatch}, {props}: { props: { uuid: string } & Partial<ISelectOneResponseBlock> }) {
     props.type = BLOCK_TYPE
     const blankPromptResource = await dispatch('flow/flow_addBlankResourceForEnabledModesAndLangs', null, {root: true})
     props.config = {
       prompt: blankPromptResource.uuid,
-      choices: {},
+      choices: [],
       minimum_choices: undefined,
       maximum_choices: undefined,
     }
     return baseActions.createWith({getters, dispatch} as ActionContext<IEmptyState, IRootState>, {props})
   },
 }
 
-function formatTestValueForUnifiedBranchingType(block: ISelectManyResponseBlock): string {
```

Comment:
deleting this for selectMany, because the tests are not clearly defined yet. (not in ALPHA scope)